src/core/server/index.ts

```diff
@@ -56,6 +57,13 @@ export class Server {
 
     await this.legacy.service.start(httpServerInfo);
 
+    if (this.env.cliArgs.open && httpServerInfo) {
+      const basePath = httpConfig.rewriteBasePath ? httpConfig.basePath : '';
+      const uri = `${httpServerInfo.server.info.uri}${basePath}`;
```

Comment:
`httpServerInfo.server.info.uri` is the wrong URL, when using `--dev` it's opening `http://localhost:5603`, but it should be opening `http://localhost:5601`

Reply:
Yeah, I'll switch that to use the `5601`